record_audio.go

```diff
@@ -224,3 +171,58 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 	}
 
 }
+
+
+
+
+//AWS - S3
+
+func ConnectAws() *session.Session {
```

Comment:
Have these function definitions in a different file. All your code is in 1 file mainly